src/Native/Unix/Common/pal_utilities.h

```diff
@@ -196,3 +197,21 @@ static inline bool CheckInterrupted(int32_t result)
 }
 
 #endif // __cplusplus
+
+inline static uint32_t UnsignedCast_Int32ToUint32(int32_t value)
```

Comment:
@bartonjs, @stephentoub do you prefer this long function names or would you like shorter ones better (e.g. just Int32ToUInt32, etc.)?

Reply:
I'd go with `Int32ToUInt32`.  It's descriptive enough.

Reply:
I've simplified the names.